beacon-chain/rpc/beacon/assignments.go

```diff
@@ -49,11 +39,12 @@ func (bs *Server) ListValidatorAssignments(
 		requestedEpoch = q.Epoch
 	}
 
-	if requestedEpoch > helpers.CurrentEpoch(headState) {
```

Comment:
Using time to detect what epoch is current instead of the head?

Reply:
Correct, that's what I updated to. See below 👇 
```go
	currentEpoch := helpers.SlotToEpoch(bs.GenesisTimeFetcher.CurrentSlot())
```